.travis.yml

```diff
@@ -2,21 +2,27 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, nightly, hhvm]
+php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
```

Comment:
I switch to 7.0 instead of nightly, so that this can use 7.0 releases once they become available on Travis. currently, it is just an alias for nightly